app/src/main/java/hous/release/android/presentation/profile/edit/ProfileEditActivity.kt

```diff
@@ -27,6 +28,29 @@ class ProfileEditActivity :
         initBirthdayOnClickListener()
         initBirthdayPublicOnClickListener()
         initSaveBtnColor()
+        initBackPressedCallback()
+    }
+
+    private fun initBackPressedCallback() {
+        onBackPressedDispatcher.addCallback(
+            this,
+            object : OnBackPressedCallback(true) {
+                override fun handleOnBackPressed() {
+                    WarningDialogFragment().apply {
+                        arguments = Bundle().apply {
+                            putSerializable(
+                                WarningDialogFragment.WARNING_TYPE,
+                                WarningType.WARNING_EDIT_PROFILE
+                            )
+                            putParcelable(
+                                WarningDialogFragment.CONFIRM_ACTION,
+                                ConfirmClickListener(confirmAction = { finish() })
+                            )
+                        }
+                    }.show(supportFragmentManager, WarningDialogFragment.DIALOG_WARNING)
```

Comment:
```suggestion
        onBackPressedDispatcher.addCallback{
            WarningDialogFragment().withArgs {
                            putSerializable(
                                WarningDialogFragment.WARNING_TYPE,
                                WarningType.WARNING_EDIT_PROFILE
                            )
                            putParcelable(
                                WarningDialogFragment.CONFIRM_ACTION,
                                ConfirmClickListener(confirmAction = { finish() })
                            )
                        }
                    }.show(supportFragmentManager, WarningDialogFragment.DIALOG_WARNING)
        }
```
- backpressd ktx를 사용해주세요!! 
- 제가 만든 `withArgs` 확장함수도 애용부탁드립니다 ㅋㅋㅋㅋ

Reply:
홍보하는거에서 빵터졌습니다 ㅋㅋㅋㅋㅋㅋㅋㅋ